squidb/src/com/yahoo/squidb/data/adapter/DefaultOpenHelperWrapper.java

```diff
@@ -53,7 +54,11 @@ public void onConfigure(SQLiteDatabase db) {
 
     @Override
     public void onOpen(SQLiteDatabase db) {
-        delegate.onOpen(new SQLiteDatabaseAdapter(db));
+        SQLiteDatabaseAdapter adapter = new SQLiteDatabaseAdapter(db);
+        delegate.onOpen(adapter);
```

Comment:
Since `onConfigure()` comes before `onOpen()` in AOSP, maybe we should keep that ordering and move this call down.